project/src/main/java/com/google/sps/data/PlacesScorerImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.data;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.maps.model.LatLng;
+
+public class PlacesScorerImpl implements PlacesScorer {
```

Comment:
I feel that this class name doesn't give us more information (and it implies that that's *the* way to implement it). How about something like "PlacesWeightScorer" or "PlacesScorerByWeights"? or something else that adds information about the way the scoring is determined?

Reply:
When there is only one implementation, that's the naming convention. If we have more than one implementation (BTW, they will probably all be by some weights) the naming will change.